pycbc/live/snr_optimizer.py

```diff
@@ -181,6 +181,23 @@ def normalize_initial_point(initial_point, bounds):
     return (initial_point - bounds[:,0]) / (bounds[:,1] - bounds[:,0])
 
 
+def init_population_array(init, parameter_count):
```

Comment:
Why are you defining a function here and only using it once?  If the same basic method is used for both the pso optimizer (optimize_pso) and for di, can't we have a function that is used by both of them?

Reply:
Also, the psd optimizer already has a function called `normalize_population`, can you not use that?